plugins/CoreHome/Controller.php

```diff
@@ -279,7 +279,7 @@ public function getRowEvolutionGraph($fetch = false, $rowEvolution = null)
     private function makeRowEvolution($isMultiRowEvolution, $graphType = null)
     {
         if ($isMultiRowEvolution) {
-            return new MultiRowEvolution($this->idSite, $this->date, $graphType);
+            return new MultiRowEvolution($this->idSite, $this->date);
```

Comment:
constructor of `MultiRowEvolution` does only have two parameters.

Reply:
Or should we adjust the constructor to accept the parameter and pass it on? Or is there any special case were using the passed (potentially `null`) value will produce a different result we can only take care of by removing the argument?

Reply:
using the same as in RowEvolution should also work

Reply:
```suggestion
            return new MultiRowEvolution($this->idSite, $this->date, $graphType);
```

With the parameter being passed through the constructor I guess we should now pass it forward from here. Let's see if the two `$graphType = null` usages will have any impact that was hidden before :eyes: